lib/egg.js

```diff
@@ -40,7 +40,7 @@ class EggApplication extends EggCore {
      */
     this.messenger = new Messenger();
 
-    this.dumpConfig();
+    this.beforeStart(() => this.dumpConfig());
```

Comment:
So this should move after `this.loader.load()` that can get all config after load.